src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.exception.*;
+import duke.task.*;
+import duke.ui.*;
+import java.util.List;
+
+public class AddCommand extends Command<List<Task>> {
```

Comment:
Add javadocs to your classes and methods other than those trivial methods. These can make reviewing your code easier!